doc/adr/0011-support-return-and-total-collateral-when-building-transactions.rst

```diff
@@ -0,0 +1,49 @@
+.. _support_return_and_total_collateral_when_building_transactions:
+
+ADR 11: Support return and total collateral when building transactions
+======================================================================
+
+Date: 2022-08-30
+
+Author(s)
+---------
+
+koslambrou <konstantinos.lambrou@iohk.io>
+
+Status
+------
+
+Proposed
+
+Context
+-------
+
+In Babbage era (available after the Vasil HF), Cardano transactions will contain new collateral related fields: "return collateral" and "total collateral" collateral.
+Return collateral (also called "collateral output") and total collateral are detailed in `CIP-40 <https://cips.cardano.org/cips/cip40>`_.
+
+In summary, return collateral is a special output (basically of type ``TxOut``) that is used to send outputs in case there is a failed phase-2 validation.
```

Comment:
```suggestion
In summary, return collateral is a special output (basically of type ``TxOut``) that becomes available in case there is a failed phase-2 validation.
```